vmaas/types.go

```diff
@@ -45,6 +45,17 @@ type Request struct {
 	UseCsaf       bool `json:"use_csaf"`
 }
 
+type CvesRequest struct {
+	CveIDs              []string `json:"cve_list"`
```

Comment:
I'd prefer to call it `Cves` not `CveIDs`. It kinda implies to me that I should expect an integer not string.